editor/src/clj/editor/pipeline/bob.clj

```diff
@@ -302,25 +310,25 @@
             ;; From BundleGenericHandler
             generate-debug-symbols? (assoc "with-symbols" "")
             generate-build-report? (assoc "build-report-html" build-report-path)
-            publish-live-update-content? (assoc "liveupdate" "true"))))
+            publish-live-update-content? (assoc "liveupdate" "yes"))))
```

Comment:
Bob's command line expects `--liveupdate` option to be either `"yes"` or `"no"`, and then `Bob/invoke` converts it to `"true"`/`"false"` and sets it as a project option. So now we need to supply `"yes"`.